### PR TITLE
Allow to specify namespace in module_options

### PR DESCRIPTION
**Pull request by raphink**

### CODE REVIEW

**Comment by raphink** [comment]
I fail to understand why the cucumber tests don't pass...

**Comment by raphink** [comment]
That's right. The first way is probablematic though, when the namespace contains slashes.

**Comment by raphink** [comment]
@ekohl since #182 also changes the README, I think I'll wait until it's merged.

**Comment by ekohl** [comment]
That's now merged :)

**Comment by raphink** [comment]
Yep, I'll have a look on Monday, I'm on weekend already!

**Comment by ekohl** [comment]
Enjoy!

**Comment by raphink** [comment]
@ekohl good for you?

File: README.md [hunk 1]
```diff
@@ -276,7 +276,9 @@ msync update -m "Commit message"
 Available parameters for modulesync.yml
 
 * `git_base` : The default URL to git clone from (Default: 'git@github.com:')
-* `namespace` : Namespace of the projects to manage (Default: 'puppetlabs')
+* `namespace` : Namespace of the projects to manage (Default: 'puppetlabs').
```
Comment (ekohl) [inline README.md:279]: Not blocking for this PR, but I think we should drop the default. Puppetlabs is no longer using modulesync so exactly 0 users benefit from this default.

**Comment by ekohl** [comment]
Do we want to squash the commits?

**Comment by raphink** [comment]
Yep, let's squash

**Pull request merged**